pegasus/sites.v3/code.org/public/educate/index.haml

```diff
@@ -8,13 +8,24 @@ title: Educator Overview
   %img.educate-banner{src: "/images/homepage/fill-945x300/girl10.jpg"}
 
 #educate-curriculum-grid
-  - DB[:cdo_educate].where(section_s:'curriculum').each do |item|
-    = view :educate_box, item: item, color: "#59b9dc"
+  - curriculum_boxes = []
```

Comment:
Is your philosophy that if we do the same task at least twice it could be a helper? So you think this merits its own helper?